### PR TITLE
feat : toast-rating

### PR DESCRIPTION
**Pull request by Kevin-Solomon**
# Toast and Rating Component

## Toast

image

[Toast CodeSandbox Link](https://lhuy6.csb.app/pages/toast/toast.html)
### Variant 1
image

### Variant @

image

[Rating CodeSandbox Link](https://lhuy6.csb.app/pages/rating/rating.html)

### CODE REVIEW

File: pages/toast/toast.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const openToast = document.querySelector('#open-toast');
+const toast = document.querySelector('.toast')
+
+
+const clearToast = (timeout)=>{
+  toast.classList.remove('show');
+  clearTimeout(timeout)
+}
+openToast.addEventListener('click',()=>{
+  console.log('sike')
```
Comment (starver20) [inline pages/toast/toast.js]: Remove console logs from code.

**Review by starver20 — APPROVED** [review]
Good to go.👍🏻

**Review by mohammed-sas — APPROVED** [review]
your code is good to merge

File: pages/toast/toast.js (hunk 1 above)
Comment (Kevin-Solomon) [reply pages/toast/toast.js]: Thank you for pointing out the mistake Amar.
I have fixed the issue.

**Pull request merged**